exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/OtelToZipkinSpanTransformer.java

```diff
@@ -36,6 +33,12 @@
  */
 final class OtelToZipkinSpanTransformer {
 
+  private static final AttributeKey<String> SERVICE_NAME = AttributeKey.stringKey("service.name");
```

Comment:
do we have any common exporter module this could be put into? Probably not a big deal, but seeing the same key repeated over and over for every exporter is a little weird.

Reply:
Sort of.. there's `:exporters:common`, which works for `:exporters:zipkin` and `:exporter:jaeger`, but not for `:exporters:jaeger-thrift`. And that would only cover the duplicate references in the exporters. The other references to `service.name` would have to use something like `:sdk:common` as the common dependency, which wouldn't be terrible but not my favorite.

We can always revisit this later since its just an implementation detail.